install-wings.sh

```diff
@@ -504,6 +579,7 @@ perform_install() {
   ptdl_dl
   systemd_file
   [ "$INSTALL_MARIADB" == true ] && install_mariadb
```

Comment:
Do you know if `install_mariadb` handles to be run twice (e.g. once for panel installation and once for Wings installation, if running both on the same machine)?